src/views/Transactions.vue

```diff
@@ -148,14 +162,22 @@ export default {
         params: { partnerId }
       })
     },
-    onScroll() {
-      const height = document.getElementById('txListElement').offsetHeight
+    onScroll(evt) {
+      if (!evt) {
+        return
+      }
```

Comment:
1. `evt` cannot be undefined, remove guard
2. Rename to `event`, the name is short enough